exporters/memory/src/in_memory_span_exporter.cc

```diff
@@ -0,0 +1,39 @@
+#include "opentelemetry/exporters/memory/in_memory_span_exporter.h"
+
+#include <iostream>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace memory
+{
+InMemorySpanExporter::InMemorySpanExporter() {}
```

Comment:
Let's add an (optional) argument for the `InMemorySpanData` buffer size.

Reply:
Done